cdap-ui/app/cdap/components/FieldLevelLineage/index.js

```diff
@@ -58,23 +59,25 @@ export default class FieldLevelLineage extends Component {
     return (
       <Provider store={Store}>
         <div className="field-level-lineage">
-          <Helmet
-            title={T.translate('features.FieldLevelLineage.title', {
-              datasetId: datasetId,
-              productName: Theme.productName,
-            })}
-          />
-          <EntityTopPanel
-            breadCrumbAnchorLabel={T.translate(`${PREFIX}.back`)}
-            title={datasetId}
-            entityType={T.translate(`${PREFIX}.entity.dataset.singular`)}
-            entityIcon="icon-datasets"
-            historyBack={true}
-          />
+          <ThemeWrapper>
```

Comment:
I don't think this is needed. The context is passed on from `main.js`. We need this unless this is a common component being used in angular side as well. If its not being used in angular then the ThemeWrapper (https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/cdap/main.js#L210) should be sufficient.

Reply:
done